roxctl/central/generate/generate.go

```diff
@@ -203,6 +203,15 @@ func createBundle(logger logger.Logger, config renderer.Config) (*zip.Wrapper, e
 func OutputZip(logger logger.Logger, config renderer.Config) error {
 	logger.InfofLn("Generating deployment bundle...")
 
+	if config.EnablePodSecurityPolicies {
+		logger.InfofLn("Central deployment bundle includes PodSecurityPolicies (PSPs). This is incompatible with Kubernetes >= v1.25.")
```

Comment:
If the message excludes the deployment name it can be extracted into a function which can be called for all components and reducing duplicated code in all three places.

```suggestion
		logger.InfofLn("Deployment bundle includes PodSecurityPolicies (PSPs). This is incompatible with Kubernetes >= v1.25.")
```

Reply:
done